call-for-papers.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Call for Papers
+permalink: /call-for-papers
+---
+
+The 16th Annual International Biocuration Conference (Biocuration 2023), supported by the ISB, invites
+attendees to consider two grand challenges for the future of biocuration and the growing biocuration
+community.
+
+First, given the widespread interest in making data, metadata, and knowledge more FAIR
+(Findable, Accessible, Interoperable, Reusable) and TRUST-worthy (Transparency, Responsibility,
+User Focus, Sustainability, and Technology), how can the community move past these as superficial
+platitudes and towards real impact?
+
+Second, given how can the community spread past its prefix of
```

Comment:
typo. remove 'given'

Reply:
Dear @deepakunni3 -- see #4 for updated text. 🌷

Reply:
Thank you for taking the time to review the text @monicacecilia ! :)